### DISCUSSION TITLE
音樂類型

### DISCUSSION THREAD

**Discussion by yuan3x2** [1101-GS4719-C/Python-Community-GS4719-C#2245] [General]
看到有同學發讀書會不會聽音樂以及最近許多歌手開演唱會，讓我蠻好奇大家平常是否會聽音樂，都聽什麼樣類型的音樂呢？
我自己是很喜歡西洋的流行樂，順便分享我自己喜歡的樂團是5SOS，最喜歡的歌手是Ruel，他寫的歌詞都很有意境跟質感，超棒的！
平常都是靠著他們撐過考試週的，可以說是心靈慰藉。

**Comment by zoo100130** [comment]
讀書聽音樂我是分兩個目的，一個是要幫助我快速進入心流;一個是放鬆心情。
想快點進入狀態的話我會聽白噪音+一點鋼琴或單傳白噪音(eg:營火燃燒、下雨聲)這類
至於放鬆心情看我當天的心裡天氣: )，聽得很雜。
有時會聽獨立音樂，就聽團類的那種
日文歌:Arekun (あれくん)的專輯、Yorushika (ヨルシカ)etc... 耳熟能詳的應該就<<所以我放棄了音樂>>
英文歌:Hazbin Hotel 裡的配樂<<You are loser>>、<<You don't know>>
有時也會聽巴哈、古典樂

Btw,到現在我還是不太會操作spotify的介面lol

**Comment by Boying12** [comment]
在我的音樂清單中，有一位歌手我特別喜歡，那就是Adele。她的音樂風格獨特，搭配她那深情且富有力量的嗓音，讓她的歌曲總是能引起我的共鳴。Adele的歌詞往往充滿情感，無論是對愛情的思考還是對生活的感慨，都讓我感受到深刻的共鳴。她的幾首經典歌曲如《Someone Like You》和《Rolling in the Deep》，每次聽到都會讓我感到心潮澎湃，仿佛她在用音樂講述我的故事。

**Comment by Boying12** [comment]
除了Adele，我也喜歡One OK Rock，他們的音樂風格融合了搖滾和流行元素，旋律非常動人，讓我在不同的情境中都有不同的情感寄託。
平常在考試週或是壓力大的時候，我總是會打開Adele的音樂，讓她的旋律陪伴我度過那些艱辛的日子。可以說，她的音樂成為了我心靈的慰藉，讓我在忙碌的生活中找到片刻的安靜與平和。音樂對我來說，不僅僅是消遣，它更像是一種情感的宣洩和抒發。

**Comment by Taoyi-Chiang** [comment]
我最喜歡的還是90年代歐陸舞曲和衍伸出來的各種音樂類型。
一邊讀書、工作或運動，一邊聽動滋尬滋真的很振奮人心欸，不過對很多人來說很吵就是了。
而且這種音樂因為節奏點非常明確的關係，所以也很好帶入需要跳舞的場合，並且因為也有大量的人聲，所以也很適合跟著唱。
推兩個樂團或歌手：Cascada、Italobrothers
https://www.youtube.com/watch?v=4G6QDNC4jPs
https://www.youtube.com/watch?v=jU_H6M4tvPw

**Comment by christaaah** [comment]
平常我都聽JPOP，但是也不排斥其他的音樂類型，最少聽的是搖滾樂和饒舌。如果遇到喜歡的音樂時會一直聽那一首歌，我只有在快要睡著但是不能的時候會聽一些搖滾樂。最近我最喜歡的音樂是vocaloid歌曲，因為這些歌的故事性很高，不同的虛擬歌手會有不一樣的風格，並且就算是同一個虛擬歌手也會因為不同P主而有不同的幹絕，讓我覺得蠻有意思的。

**Comment by Mminniehuang** [comment]
我會在不同場合聽不同類型的歌，比如說在讀書當下我會選擇鋼琴音樂，因為流行歌經常會讓我分心甚至開始唱歌，當然流行歌（不論是西洋或是華語我都蠻喜歡聽的）我會在想要舒壓的時候，或者坐車的時候聽，有時候滑著youtube時也會隨機點幾首就開始聽起來了。

**Comment by yue083139** [comment]
我平常喜歡聽J-pop，最初是國中的時候聽了vocaloid的歌曲覺得很新穎有趣就入坑了，不知不覺也就開始聽一些其他的歌手所創作的歌，喜歡這些歌主要是因為歌曲背後通常有些有趣的故事，有時也會探討一些特定的議題，給單純的聽音樂提升了一個層次，最近喜歡聽tuki 的サクラキミワタシ，但無論是哪種音樂能夠讓自己得到寄託就是好音樂。

**Comment by 113607514** [comment]
我也是覺得音樂真的是一個很好的心靈慰藉，尤其是在忙碌的考試週。有時候聽著5SOS或Ruel的歌，會讓整個人放鬆下來，重新充電。其實音樂不僅是消遣，對我來說，它常常能帶來靈感，也能幫我舒解壓力。你平常還會聽些什麼類型的音樂呢？

**Comment by Taoyi-Chiang** [comment]
有一陣子很喜歡地球音的類型，雖然好像不太算音樂，但睡午覺的時候開著海浪或潮汐的聲音，其實感覺滿療癒的。

**Comment by Elaine4488** [comment]
我蠻喜歡抒情類型的歌，大概是因為那種平靜的旋律能讓我在忙碌中找到一點喘息的空間。尤其是考試壓力大的時候，抒情歌就像一種陪伴，可以讓我稍微停下來，理清自己的情緒。歌詞也很重要，那些能引起共鳴的詞，常常會讓我想更多，有時候甚至幫助我看事情的角度變得不一樣。

**Comment by 113607514** [comment]
每次考試週一到，我的耳機幾乎不離身。當別人埋頭刷題，我就開著5SOS的歌單，讓他們的音樂在背景裡陪著我。那種感覺就像有幾個老朋友在旁邊，一起熬夜打怪，雖然現實是我對著課本苦戰（笑）。  

Ruel的歌更是救命稻草，有時候壓力大到想放棄，只要聽到他的聲音，整個人瞬間安靜下來。那些細膩的歌詞，好像總能精準地戳中我內心最疲憊的地方，但又帶來一絲安慰。可以說，他的歌就是我每次快崩潰時的一道微光。  

不知道大家是不是也有這種「考試專屬歌單」？音樂真的有一種魔力，讓人即使身處混亂，也能找到一點平靜。

**Comment by qiu0109** [comment]
近幾年很喜歡聽JPOP，當初考分科的時候也是聽著JPOP讀書的，我覺得音樂最大的優點就是可以讓人感到不無聊，沒有音樂的話我就沒有辦法坐在書桌前好幾個小時專注的讀書。

**Comment by poyen0806** [comment]
- [心做し](https://youtu.be/QJie7dTvbjQ?si=IUjPTEprOjhVc6LJ)
- [絆](https://youtu.be/IxYO5v7DCTM?si=X1ae3CH5Om8Xh5pU)
- [僕が死のうと思ったのは](https://youtu.be/AabAbM_Lmuo?si=iMbETW3Vp04zTOJT)
- [Unravel](https://youtu.be/Fve_lHIPa-I?si=UKkeUpPeRBykq5fn)
- [壱雫空](https://youtu.be/cjyxTk_IfIY?si=hcma8nlciurBUv8y)
- [迷星叫](https://youtu.be/qPv7-zXpq4M?si=n2_aRk4QZMNG0M6q)
- [春日影](https://youtu.be/-kZBuzsZ7Ho?si=_MXWI21Hq0XdOS2O)
- [詩超絆](https://youtu.be/wWf9k4e2xEo?si=TVbndxWwvBIkYIUP)

我喜歡聽日文歌，以上是我覺得不錯聽的，後來去看日文歌詞後覺得別有一番風味。
